components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -232,9 +238,10 @@ input ConfigEntryInput {
 input ComponentConfigurationInput {
     component: String!                    # Kyma component name
     namespace: String!                    # Namespace to which component should be installed
+    prerequisite: Boolean                 # Specifies if component is prerequisite or it is optional
```

Comment:
We should also add prerequisite field to KymaConfig type to be able to return it in `runtimeStatus` query.